demisto_sdk/commands/download/downloader.py

```diff
@@ -216,23 +299,23 @@ def build_custom_content(self) -> None:
         Build a data structure called pack content that holds basic data for each content entity instances downloaded from Demisto.
         For example check out the CUSTOM_CONTENT variable in downloader_test.py
         """
-        # Build custom content files
-        custom_content_file_paths: list = get_child_files(self.custom_content_temp_dir)
-        custom_content_objects: list = list()
-        for custom_content_file_path in custom_content_file_paths:
-            custom_content_objects.append(self.build_custom_content_object(custom_content_file_path))
-
         for input_file_name in self.input_files:
             input_file_exist_in_cc: bool = False
-            for custom_content_object in custom_content_objects:
+            for custom_content_object in self.get_custom_content_objects():
```

Comment:
If the `-a` flag is up this process is happening twice? It is happening also on line 185 any chance we can reuse it?

Reply:
Yes, refactored